desktop/core/src/desktop/js/components/sidebar/AccordionItem.vue

```diff
@@ -29,6 +29,7 @@
   >
     <button
       class="sidebar-base-btn sidebar-sidebar-item"
+      data-testid="siderbar--query-editor--button"
```

Comment:
This is not (only) the query editor button, it needs a condition to set the id depending on the item.

Reply:
By item here we mean?

Reply:
A menu item, it's generic so not only for the editor. If you were to query for this data test id I bet you'd get multiple in the result, it has to be unique.